web/themes/custom/pineapple/sass/components/paragraphs/_image-paragraph.scss

```diff
@@ -1,8 +1,19 @@
 .paragraph--type--image-paragraph {
-  .image-paragraph-image {
-    img {
-      object-fit: cover;
-      width: 100%;
+  .image-paragraph-container {
```

Comment:
What's this class for? it doesn't do anything

Reply:
done